rules/style.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const isDev = process.env.NODE_ENV === 'development';
+
+/** @type {import('@types/eslint').Linter.Config} */
+module.exports = {
+  rules: {
+    strict: ['error', 'safe'],
+    indent: ['error', 2, {
+      SwitchCase: 1,
+      MemberExpression: 'off'
+    }],
+    'arrow-parens': ['error', 'as-needed'],
+    'prefer-const': 'on',
```

Comment:
Any rule that is already defined as part of `semi/standard` and we are not overriding it should be removed from list.